backend/src/folder.ts

```diff
@@ -0,0 +1,127 @@
+import { APIGatewayEvent, Context, ProxyResult } from 'aws-lambda'
+const Folder = require('../model/Folder')
+const querystring = require('querystring')
```

Comment:
querystring is does what? is that native to the NodeJS?

Reply:
Yes, it is native to NodeJS. It parses event.body in x-www-form-urlencoded format